src/ert/gui/simulation/run_dialog.py

```diff
@@ -250,9 +252,6 @@ def __init__(
         self.running_time = QLabel("")
         self.queue_system = QLabel("")
         self.memory_usage = QLabel("")
-        self.disk_space = DiskSpaceWidget(
-            get_mount_directory(self._run_model_api.runpath_format_string)
```

Comment:
Did you undderstand why this property of the _run_model_api was erroneous? Is it another bug?

Reply:
https://github.com/equinor/ert/pull/10859#discussion_r2088904995